scripts/devcloud-daily.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
+
+# SPDX-FileCopyrightText: Intel Corporation
+#
+# SPDX-License-Identifier: BSD-3-Clause
+set -e
```

Comment:
I moved the `set -e` down below the `source /opt..` in the other scripts to make the log more readable.

Reply:
Done